File: curso_python3/Matriz.py
    def imprimirMatriz(matriz):
    for contad in range(0, 3, +1):    
        for con in range(0, 3, +1):
            print(matriz[contad][con], end = ' ')
        print()

def inserirElementos(matriz):
    for c in range(0, 3, +1):
        for cont in range(0, 3, +1):
            try:
                elemento = int(input('Digite um elemento na linha ' + str(c + 1) + ' coluna ' + str(cont + 1) + ' da matriz: '))
                matriz[c][cont] = elemento
            except ValueError:
                print('elemento digitado inválido')
                matriz[c][cont] = 0

def main():
    print('*** Matriz ***')

    matriz = [[None, None, None],[None, None, None],[None, None, None]]
    matriz2 = [[None, None, None],[None, None, None],[None, None, None]]
    matriz3 = [[None, None, None],[None, None, None],[None, None, None]]

    inserirElementos(matriz)
    inserirElementos(matriz2)

    for c in range(0, 3, +1):
        for cont in range(0, 3, +1):
            matriz3[c][cont] = matriz[c][cont] + matriz2[c][cont]
            
    matrizes = [matriz, matriz2, matriz3]
    for contad in range(0, 3, +1):
        print('*** Matriz ' + str(contad + 1) + ' ***')
        imprimirMatriz(matrizes[contad])

main()

File: Curso_C/Aulas/Matriz.c
#include <stdio.h>

int main() {

    int matriz[3][3];

    matriz[0][0] = 7;
    matriz[0][1] = 6;
    matriz[0][2] = 9;
    
    matriz[1][0] = 5;
    matriz[1][1] = 1;
    matriz[1][2] = 9;
    
    matriz[2][0] = 5;
    matriz[2][1] = 5;
    matriz[2][2] = 10;

    for (int c = 0; c < 3; c++) {
        for (int cont = 0; cont < 3; cont++) {
            printf("%i ", matriz[c][cont]);
        }
        printf("\n");
    }
}
        
